lms/resources/_js_config/__init__.py

```diff
@@ -68,6 +69,16 @@ def add_document_url(self, document_url):
                     resource_link_id=self._request.lti_params["resource_link_id"],
                 ),
             }
+        elif document_url.startswith("d2l://"):
+            self._config["api"]["viaUrl"] = {
+                "authUrl": self._request.route_url(D2L.route.oauth2_authorize),
+                "path": self._request.route_path(
+                    "d2l_api.courses.files.via_url",
+                    course_id=self._request.lti_params["context_id"],
+                    _query={"document_url": document_url},
+                ),
+            }
```

Comment:
Ah ok, this is the config telling the frontend to make the API call to get the Via URL? :+1: